### PR TITLE
Fix fullscreen when tabing out

### PR DESCRIPTION
**Pull request by Alexdoru**
fix the vanilla 1.7.10 bug that makes your game exit fullscreen when the game looses focus (if you use alt tab)

tested and working in full pack

### CODE REVIEW

File: src/main/java/com/mitchej123/hodgepodge/mixins/Mixins.java [hunk 1]
```diff
@@ -71,10 +71,15 @@ public enum Mixins {
             () -> Hodgepodge.config.fixGlStateBugs,
             TargetedMod.VANILLA),
     FIX_FULLSCREEN_RESIZABLE(
-            "minecraft.MixinMinecraft",
+            "minecraft.MixinMinecraft_fixFullscreenResizable",
```
Comment (Glease) [inline src/main/java/com/mitchej123/hodgepodge/mixins/Mixins.java:74]: I'm fine with either of prepending `fix` or not prepending `fix`, but not mixing them

Comment (Alexdoru) [reply src/main/java/com/mitchej123/hodgepodge/mixins/Mixins.java:74]: You are right, my mistake I copy pasted the names too fast. I'll remove the fix

**Comment by chochem** [comment]
Alternatively we could just consider including/forking this mod: https://github.com/hancin/Fullscreen-Windowed-Minecraft/tree/mc-1.7.10-backport

I have used it for years and we have had it as a recommended extra mod. The license is BSD-2-Clause, so that should be fine too.


At least in my opinion that is the much better solution.

**Comment by boubou19** [comment]
We have enough mods that does fix/optimisations. It's time to unify all of those under some existing mods.

**Comment by chochem** [comment]
yea I get that.

File: src/main/java/com/mitchej123/hodgepodge/mixins/Mixins.java (hunk 1 above)
Comment (Alexdoru) [reply src/main/java/com/mitchej123/hodgepodge/mixins/Mixins.java]: done

**Review by Glease — APPROVED** [review]

**Pull request merged**